src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -1,19 +1,19 @@
   "lambdynlights.option.block_entities": "Entità blocco",
   "lambdynlights.option.entities": "Entità",
   "lambdynlights.option.mode": "Luci Dinamiche",
-  "lambdynlights.option.water_sensitive": "Controllo di sensibilità dell'acqua",
-  "lambdynlights.toast.first_time": "Disattivato di predefinito, controlla impostazioni della mod o di Grafica!",
+  "lambdynlights.option.water_sensitive": "Controllo della sensibilità dell'acqua",
+  "lambdynlights.toast.first_time": "Disattivata di predefinito, controlla le impostazioni grafiche o della mod!",
   "lambdynlights.tooltip.block_entities": "Attiva l'illuminazione dinamica delle entità blocco.",
-  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità di illuminazione dinamica dei Creeper.\n- %s disattiva l'illuminazione dinamica dei Creeper.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
-  "lambdynlights.tooltip.entities": "Attiva l'illuminazione dinamica di entità. Nota: I giocatori sono sempre attivi.",
-  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, un giocatore con una torcia in mano emetterà luce per esempio.",
-  "lambdynlights.tooltip.mode.2": "%s e %s avranno un leggero ritardo ma minimizzeranno gli aggiornamenti di luce.",
-  "lambdynlights.tooltip.mode.3": "%s è illuminazione dinamica soffusa.",
-  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità di illuminazione dinamica della TNT.\n- %s disattiva l'illuminazione dinamica della TNT.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
-  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo di fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti in mano non emetteranno luce mentre sommersi in acqua."
+  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità dell'illuminazione dinamica dei Creeper.\n- %s disattiva l'illuminazione dinamica dei Creeper.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
+  "lambdynlights.tooltip.entities": "Attiva l'illuminazione dinamica delle entità. Nota: l'illuminazione dinamica dei giocatori è sempre attiva.",
+  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, per esempio, un giocatore con una torcia in mano emetterà luce.",
+  "lambdynlights.tooltip.mode.2": "%s e %s avranno un leggero ritardo ma minimizzeranno gli aggiornamenti della luce.",
+  "lambdynlights.tooltip.mode.3": "%s è attiva l'illuminazione dinamica soffusa.",
+  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità d'illuminazione dinamica della TNT.\n- %s disattiva l'illuminazione dinamica della TNT.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
+  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo delle fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti non emetteranno luce mentre sommersi in acqua."
```

Comment:
Forse quell'"in acqua" finale è un po' rindondante, che ne pensi?

Reply:
Già, il sommersi indica già che si tratta di acqua quindi si può anche rimovere la specificazione di ciò.